netutils/vlan.py

```diff
@@ -28,14 +33,16 @@ def vlanlist_to_config(vlan_list, first_line_len=48, other_line_len=44, min_grou
         ['1,3,5,6,100,101,102,103,104,105,107,109']
     """
 
-    def build_final_vlan_cfg(vlan_cfg):
+    def build_final_vlan_cfg(vlan_cfg: str) -> t.List[str]:
         if len(vlan_cfg) <= first_line_len:
             return [vlan_cfg]
 
         # Split VLAN config if lines are too long
         first_line = re.match(f"^.{{0,{first_line_len}}}(?=,)", vlan_cfg)
+        if not first_line:
+            raise ValueError(f"Parsing VLAN list with first_line_len={first_line_len} yielded no regex results.")
```

Comment:
@jeffkala I'm going to need your help on this one because you understand the business logic better, can you give a suggestion to make this error message a little friendlier to everyone? I think not even the software engineer will be able to do something with this 🤦🏻